test/runtests.jl

```diff
@@ -335,16 +354,16 @@ import TextParse: _csvread
     """
     data = ((["x", "","x","x y","x"],
               ones(5),
-              DataValueArray(ones(5), Bool[0,1,0,0,1]),
-              DataValueArray(ones(Int,5), Bool[0,0,0,1,0])),
+              [1,missing,1,1,missing],
```

Comment:
`[1.0,missing,1.0,1.0,missing]`? Just to technically match what it was previously.